### PR TITLE
Simplify assessment metrics into a simple script

### PR DESCRIPTION
**Pull request by claudijd**

### CODE REVIEW

File: security_assessment_products/generate_assessment_metrics.py [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+# How many vulnerability assessments are invalid?
+invalid_vas = 0
+
+for bug in va_metrics:
+    if bug['status'] == "RESOLVED" and bug['resolution'] == "INVALID":
+        invalid_vas += 1
+
+print("- {} vulnerability assessments were invalid.".format(invalid_vas))
+
+# How many vulnerability assessments have are in progress?
```
Comment (caggle) [inline security_assessment_products/generate_assessment_metrics.py]: "have" is unnecessary

File: security_assessment_products/generate_assessment_metrics.py [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+# How many vulnerability assessments are invalid?
+invalid_vas_60 = 0
+
+for bug in va_metrics_60:
+    if bug['status'] == "RESOLVED" and bug['resolution'] == "INVALID":
+        invalid_vas_60 += 1
+
+print("- {} vulnerability assessments were invalid.".format(invalid_vas_60))
+
+# How many vulnerability assessments have are in progress?
```
Comment (caggle) [inline security_assessment_products/generate_assessment_metrics.py:240]: Ditto

**Comment by caggle** [comment]
Looks good to me.

File: security_assessment_products/generate_assessment_metrics.py (hunk 1 above)
Comment (claudijd) [reply security_assessment_products/generate_assessment_metrics.py]: fixed

File: security_assessment_products/generate_assessment_metrics.py (hunk 2 above)
Comment (claudijd) [reply security_assessment_products/generate_assessment_metrics.py:240]: fixed

**Pull request merged**